snapshot/store/http_store.go

```diff
@@ -46,11 +46,21 @@ type httpStore struct {
 	client  Client
 }
 
-func (s *httpStore) OpenForRead(name string) (io.ReadCloser, error) {
+func getTTLValue(resp *http.Response) *TTLValue {
+	ttlk := http.CanonicalHeaderKey("x-ton-expires")
```

Comment:
If we do this we should use the DefaultTTLKey in snapshot/store/store.go as the default, and if the httpStore config has a TTLConfig with TTLKey specified, to use that instead (so this will work with internal scoot)

Reply:
I might be missing something, but It does not appear that the http store is configurable. Should I make it configurable?

Reply:
Yep, you are right, will need to add TTLConfig